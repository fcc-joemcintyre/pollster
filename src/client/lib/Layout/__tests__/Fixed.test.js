/* eslint-env jest */
import React from 'react';
import renderer from 'react-test-renderer';
import { Fixed } from '..';

test ('Fixed renders default component', () => {
  const component = renderer.create (
    <Fixed />
  );
  const tree = component.toJSON ();
  expect (tree).toMatchSnapshot ();
});

test ('Fixed renders positioned component', () => {
  const component = renderer.create (
    <Fixed top={10} left={10} bottom={10} right={10} />
  );
  const tree = component.toJSON ();
  expect (tree).toMatchSnapshot ();
});
