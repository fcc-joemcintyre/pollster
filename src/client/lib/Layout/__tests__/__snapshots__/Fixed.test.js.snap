// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Fixed renders default component 1`] = `
.c0 {
  position: fixed;
  z-index: 100;
}

<div
  className="c0"
/>
`;

exports[`Fixed renders positioned component 1`] = `
.c0 {
  position: fixed;
  top: 10;
  left: 10;
  right: 10;
  bottom: 10;
  z-index: 100;
}

<div
  className="c0"
/>
`;
